utils/hct/hctbuild.cmd

```diff
@@ -184,13 +179,19 @@ if "%1"=="-dxc-cmake-ends-include" (
   shift /1
 )
 
-rem If only VS 2017 is available, pick that by default.
-if "%BUILD_VS_VER%"=="2015" (
-  reg query HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\DevDiv\vs\Servicing\14.0\devenv /v Install /reg:32 1>nul 2>nul
-  if errorlevel 1 (
-    echo Visual Studio 2015 not available, setting up build for Visual Studio 2017.
-    set BUILD_GENERATOR=Visual Studio 15 2017
-    set BUILD_VS_VER=2017
+rem If only VS 2019 is available, pick that by default.
+if "%BUILD_VS_VER%"=="2017" (
+  if not exist "%ProgramFiles(x86)%\Microsoft Visual Studio\2017" (
+    echo Visual Studio 2017 not available, setting up build for Visual Studio 2019.
```

Comment:
Since this results from a user specifying 2017, it might be more accurate to say "trying to set up build" since it might well fail since they might have specified 2017 since they don't have 2019.

I'm not sure I would find this behavior helpful. Covertly getting 2019 when I specified 2017 might confuse and ultimately frustrate me more than being told it wasn't available and I'd have to use 2019.

Reply:
Good point, I'll fix that.